service/test/controllers/notesmanagement/validator/NoteRequestValidatorTest.java

```diff
@@ -0,0 +1,93 @@
+package controllers.notesmanagement.validator;
+
+import controllers.BaseApplicationTest;
+import controllers.DummyActor;
+import modules.OnRequestHandler;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.request.HeaderParam;
+import org.sunbird.common.request.Request;
+
+import java.util.*;
+
+@PrepareForTest(OnRequestHandler.class)
+public class NoteRequestValidatorTest extends BaseApplicationTest {
+
+    public static Map<String, List<String>> headerMap;
+    private NoteRequestValidator noteRequestValidator;
+    private Request request;
+
+    @Before
+    public void before() {
```

Comment:
can we not define before() method inside BaseAppliation and child will used it. we don't need to clear map inside tearDown. becuase once test case done jvm will be clean it automatically

Reply:
done